impl/src/main/java/org/jboss/weld/bootstrap/AnnotatedTypeLoader.java

```diff
@@ -42,6 +42,7 @@
     final ContainerLifecycleEvents containerLifecycleEvents;
 
     static final String MODULEINFO_CLASS_NAME = "module-info";
+    static final String MULTI_RELEASE_JAR_PATH = "META-INF.versions";
```

Comment:
Hm, so a class from `META-INF/versions` has a name like `META-INF.versions.9.org.acme.Foo`?

Reply:
Forgot to mention that, sorry. Yeah, within the `loadClass(String className)` method, you will see that class as something like this: `META-INF.versions.12.io.smallrye.context.impl.JdkSpecificImpl`

Reply:
@mkouba are you OK with this PR or should I change something?